src/aleph_client/__init__.py

```diff
@@ -8,3 +8,12 @@
     __version__ = "unknown"
 finally:
     del get_distribution, DistributionNotFound
+
+# Deprecation check
+moved_types = ["AlephClient", "AuthenticatedAlephClient", "synchronous", "asynchronous"]
+
+
+def __getattr__(name):
+    if name in moved_types:
+        raise ImportError(f"The 'aleph_client.{name}' type is deprecated and has been removed from aleph_client. Please use `aleph.sdk.{name}` instead.")
```

Comment:
`ImportError` does not seem like the right exception here :thinking: 
[See https://docs.python.org/3/library/exceptions.html](https://docs.python.org/3/library/exceptions.html#exception-hierarchy)
`AttributeError`  is the standard class for this in Python, see 
https://docs.python.org/3/reference/datamodel.html#object.__getattr__

Reply:
But actually, ImportError is fine: https://docs.python.org/3/library/exceptions.html#ImportError

Reply:
I don't agree, this is not an import error. It is not "the [import](https://docs.python.org/3/reference/simple_stmts.html#import) statement has troubles trying to load a module"